AppOne/AppOne/DataServiceLayer/ServiceLayer.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ServiceLayer.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 13.12.22.
+//
+
+import Foundation
+
+protocol ServiceCalls {
+    func fetchToDOs(completion: @escaping (Result<[ToDos], ErrorType>) -> Void)
+    func fetchUsers() async throws -> [Users]
+    func fetchPhotos() async throws -> [Photos]
+}
+
+class ServiceLayer: ServiceCalls {
```

Comment:
Also I would do one service per model -> like Todo and User and Photo or make it generic. But we can do this later